scripts/find-mem-leak.cocci

```diff
@@ -2,8 +2,8 @@
 expression x, y;
 statement S;
 @@
-  x = mbedtls_malloc(...);
-  y = mbedtls_malloc(...);
+  x = mbedtls_calloc(...);
+  y = mbedtls_calloc(...);
   ...
```

Comment:
@andresag01 Ok. If supported by the semantic match language, one might restrict `S` to statements not involving `x` and `y` - but that's an issue independent from the switch to `calloc`.